modules/hal_nordic/nrfx/nrfx_config.h

```diff
@@ -564,6 +564,22 @@
 #define NRFX_UARTE3_ENABLED 1
 #endif
 
+#ifdef CONFIG_NRFX_UARTE_CONFIG_SKIP_GPIO_CONFIG
+#define NRFX_UARTE_CONFIG_SKIP_GPIO_CONFIG 1
+#endif
+
```

Comment:
These blank lines are inconsistent with what we have for other drivers.
And actually these translations and the Kconfig options themselves should be introduced in a separate commit.